api/python/provisioner/inputs.py

```diff
@@ -314,7 +314,10 @@ def fill_parser(cls, parser, *args, **kwargs):
 
     @classmethod
     def from_args(cls, parsed_args, *args, **kwargs):
-        return ParserFiller.from_args(cls, parsed_args, *args, **kwargs)[0]
+        if cls is ParserFiller:
```

Comment:
`if cls is ParserFiller:`

Should this be?
`if cls is not ParserFiller:`

As else is operating on same class.

Reply:
Thanks. It's a bug but a bit different, should be:
```
if cls is ParserMixin
```
and it mean that we don't expect parsing for the mixin itself